src/scripts/modules/media/header/header-template.html

```diff
@@ -14,15 +14,12 @@
     {{/if}}
 
     <div class="title">
-      <h2 data-l10n-id="textbook-view-header-{{#isnt status 'publishing'}}no-{{/isnt}}publishing{{#if chapter}}-chapter{{/if}}" data-l10n-args='{"chapter":"{{chapter}}","title":"{{safeQuotes pageTitle}}"}'>
-        {{#is status 'publishing'}}
-          <span class="label label-info">publishing</span>
-        {{/is}}
+      <h2>
         {{#if chapter}}<span class="title-chapter">{{chapter}}</span>{{/if}}
         {{{pageTitle}}}
```

Comment:
I don't know why the Page title is not highlighted... I switched this code to not go through l20n so it _should_ output highlighted...